.github/workflows/lib-ms.yml

```diff
@@ -85,7 +83,7 @@ jobs:
           cache: "yarn"
           cache-dependency-path: "**/yarn.lock"
           always-auth: true
-          registry-url: 'https://npm.pkg.github.com/'
+          registry-url: "https://npm.pkg.github.com/"
```

Comment:
please retain the single quotes

Reply:
It's the auto formatting...

Reply:
You can add

```json
  "prettier": {
    "singleQuote": true
  },
````

to package.json. See the [example](https://github.com/INTO-CPS-Association/DTaaS/blob/feature/distributed-demo/client/package.json#L32C1-L35C5)